src/location.c

```diff
@@ -524,16 +524,16 @@ handle_start_in_thread_func (GTask *task,
 
       g_variant_builder_init (&access_opt_builder, G_VARIANT_TYPE_VARDICT);
       g_variant_builder_add (&access_opt_builder, "{sv}",
-                             "deny_label", g_variant_new_string (_("Deny Access")));
+                             "deny_label", g_variant_new_string (_("Deny")));
       g_variant_builder_add (&access_opt_builder, "{sv}",
-                             "grant_label", g_variant_new_string (_("Grant Access")));
+                             "grant_label", g_variant_new_string (_("Allow")));
       g_variant_builder_add (&access_opt_builder, "{sv}",
                              "icon", g_variant_new_string ("find-location-symbolic"));
 
       if (g_str_equal (app_id, ""))
         {
-          title = g_strdup (_("Grant Access to Your Location?"));
-          subtitle = g_strdup (_("An application wants to use your location."));
+          title = g_strdup (_("Allow Access to Your Location?"));
```

Comment:
```suggestion
          title = g_strdup (_("Allow an Application Access to Your Location?"));
```

Reply:
I think this is wrong. Since `g_str_equal (app_id, "")` we are granting the permission to all non-identified applications, not just one.